text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala

```diff
@@ -121,6 +121,17 @@ class OdinActions(val taxonomy: Taxonomy, expansionHandler: Option[ExpansionHand
     mentions.flatMap(mkDefinitionMention)
   }
 
+  def lookLikeAVariableInAnyRel(mentions: Seq[Mention], state: State): Seq[Mention] = {
```

Comment:
Instead, can you modify the base version with something like words = m match {...}